src/nvme/tree.h

```diff
@@ -469,7 +469,7 @@ nvme_ns_t nvme_subsystem_next_ns(nvme_subsystem_t s, nvme_ns_t n);
  *
  * Return: File descriptor associated with @n or -1
  */
-int nvme_ns_get_fd(nvme_ns_t n);
+void* nvme_ns_get_fd(nvme_ns_t n);
```

Comment:
The function name should also be updated alongside. Also I'd like to see a proper type returned as handle.